NU394-S2024TTh/tribe-a
# added reviews input box

### Description

Added box to paste in reviews.

### QA
Uncomment console log statement on line 15 of reviewsinput.tsx.
Run app and paste in reviews separated by new lines, then click submit button.

Example text to paste in:
Great movie!
Mid as hell.
Eh, it was okay.
Fell asleep…
Worst movie ever!!!

Check console to see that it properly separated the text into an array of reviews.

### Packages
N/A

### Linear Issue Tracking

Fixes BAC-18

### Checklist
